app/src/scripts/common/partials/happybrowser.jsx

```diff
@@ -56,12 +57,16 @@ export default class Happybrowser extends React.Component {
 
   _incompatibleBrowser(ua) {
     const check = bowser.check(
-      { chrome: '49', chromium: '49', googlebot: '0' },
+      { chrome: '49', chromium: '49', googlebot: '0', firefox: '50' },
       true,
       ua || window.navigator.userAgent,
     )
     if (
-      bowser.check({ chrome: '41' }, true, ua || window.navigator.userAgent)
+      bowser.check(
```

Comment:
This check should not include Firefox, it is a fallback for Googlebot (Chrome 41 only).